tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -330,13 +332,16 @@ generate_qemu_options() {
 	# building QEMU.
 	case "$arch" in
 	aarch64)
+		qemu_options+=(functionality:--disable-virtiofsd)
```

Comment:
Maybe worth mentioning the motivation (we've switched to Rust implementation) in the commit message ?

Reply:
Done, thanks!